src/PresentationalComponents/Charts/ActionsOverviewDonut.js

```diff
@@ -85,41 +86,51 @@ class AdvisorOverviewDonut extends React.Component {
     };
 
     render () {
+        const { className } = this.props;
         const label = <svg
             className="chart-label"
             height={ 1 }
         >
             <ChartLabel
-                style={ { fontSize: 20 } }
-                text={ this.props.category.length ? `${this.props.category.reduce((sum, curr) => sum + curr)} Issues` : '' }
+                style={ { fontSize: 25 } }
```

Comment:
[should probably keep font size at 20 ](https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-charts/src/components/DonutChart/examples/SimpleChart.js#L32)